app/views/casework/crime_applications/show.html.erb

```diff
@@ -15,6 +15,9 @@
     <%= render partial: 'offences', object: @crime_application.case_details.offences %>
     <%= render partial: 'codefendants', object: @crime_application.case_details.codefendants %>
     <%= render partial: 'interests_of_justice', locals: { crime_application: @crime_application } %>
+    <% if FeatureFlags.evidence_upload.enabled? %>
```

Comment:
Might not be necessary as the whole partial is conditional on whether `supporting_evidence.present?` which will always be false until some provider start uploading files?

Reply:
Yeah agreed, this shouldn't be displayed as it's not being used yet.  But it was part of the ticket to be behind a feature flag so we followed that for now